plasmapy/diagnostics/tests/test_langmuir.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for functions that use Langmuir analysis functions."""
+
+import numpy as np
+
+from ..langmuir import (swept_probe_analysis,
```

Comment:
`from plasmapy.langmuir import...` would probably fix Travis screaming about this line